python/Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN cd / && \
 
 FROM clearlinux/os-core:latest
 
+ENV NAME="clearlinux-python"
 COPY --from=builder /install_root /
 
+RUN set -e echo "IMAGE NAME: $NAME"
```

Comment:
It only displayes during the docker image build.
What do you want from this log?

Reply:
should be run in docker-entrypoint.sh which is not necessary for python container. will skip this.